modules/veteran_verification/spec/requests/mock_veteran_status_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'emis/mock_veteran_status_service'
+
+describe EMIS::MockVeteranStatusService do
+  let(:edipi_veteran) { '1068619536' }
+  let(:edipi_non_veteran) { '1140840595' }
+  let(:bad_edipi) { '595' }
+  let(:missing_edipi) { '1111111111' }
+  let(:no_status) { '1005079361' }
+
+  before do
+    Settings.vet_verification.mock_emis = true
+    Settings.vet_verification.mock_emis_host = 'https://vaausvrsapp81.aac.va.gov'
+  end
```

Comment:
so, weird thing with specs and the `before` case, is that it can stay like this for the rest of the tests (no idea why) so for the non-mock tests, you'll need to add a 
```
before do
    Settings.veteran_verification.mock_emis = false
end
```
we found this out the hard way with mock-bgs deployment

Reply:
oh good call.